troubleshooting/tools/tracevol.py

```diff
@@ -194,10 +195,9 @@ def check_pv_name_in_rados(arg, image_id, pvc_name, pool_name, is_rbd):
     if arg.toolboxdeployed is True:
         kube = get_cmd_prefix(arg)
         cmd = kube + cmd
-    out = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                           stderr=subprocess.STDOUT)
+    with subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT) as out:
```

Comment:
```suggestion
    
    with subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT) as out:
```

Reply:
Leave a new line before just like with all other similar calls to make things uniform?